lib/malwareScanner.js

```diff
@@ -17,16 +17,19 @@ async function scanRequest(Attachments, key) {
 
   let currEntity = draftEntity == undefined ? activeEntity : draftEntity
 
-  await updateStatus(AttachmentsSrv, key, "Scanning", currEntity, draftEntity, activeEntity)
-
   if (!scanEnabled) {
-    setTimeout(async () => {
-      DEBUG?.('Malware scanning is disabled. Setting scan status to Clean.')
-      await updateStatus(AttachmentsSrv, key, "Clean", currEntity, draftEntity, activeEntity)
-    }, 5000)
-
-    return
+    if (cds.env.profiles.length === 1 && cds.env.profiles[0] === "development") {
+      setTimeout(async () => {
+        DEBUG?.('Malware scanning is disabled. Setting scan status to Clean in development profile.')
+        await updateStatus(AttachmentsSrv, key, "Clean", currEntity, draftEntity, activeEntity)
+      }, 5000)
+      return
+    } else {
+      return
+    }
   }
+
+  await updateStatus(AttachmentsSrv, key, "Scanning", currEntity, draftEntity, activeEntity)
```

Comment:
We can set `scanning` before `clean` also, as to replicate

Reply:
done